src/styles/index.css

```
@@ -0,0 +1,36 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  min-height: 100vh;
+  height: auto;
+  background-color: #f8f8fa;
+}
+
+a,
+a:link,
+a:visited,
+a:hover {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+/* mobile */
+#root {
+  position: relative;
+  width: 430px;
+  height: 100%;
+  margin: 0 auto;
+  background-color: white;
+  box-shadow: 0px 0px 0.315rem rgba(0, 0, 0, 0.25);
+}
```

mobile 환경에서, 고정된 width를 사용하면 다양한 기기에 대응할 수 없더라구요 ㅜ
예를 들어, 425px 넓이인 아이폰 xs에서는 5px 좌우 스크롤이 생기고, 375px 넓이인 아이폰 7,8에서는 55px 좌우로 스크롤이 생기게 돼요

```suggestion
#root {
  position: relative;
  width: 100%;
  max-width: 430px;
  height: 100%;
  margin: 0 auto;
  background-color: white;
  box-shadow: 0px 0px 0.315rem rgba(0, 0, 0, 0.25);
}
```

 저는 max-width를 걸어두고 width를 주는 편입니다!

---

진짜,, css가 젤 어려운듯요.. 감사함다